src/main.rs

```diff
@@ -90,6 +93,19 @@ fn main() {
                      clearing disk caches, for example.",
                 ),
         )
+        .arg(
+            Arg::with_name("style")
+                .long("style")
+                .short("s")
+                .takes_value(true)
+                .value_name("OPT")
```

Comment:
What does `OPT` stand for? Maybe change this to `TYPE`?

Reply:
It was chosen to represent "Option" while staying in line with your existing 3 letter notation for placeholders in the docs (NUM, CMD, etc.) as I didn't want to stray from a current convention.

Reply:
That's okay. 3 letters was just a coincidence